data/pattern-lab-categories.yml

```diff
@@ -0,0 +1,34 @@
+categories:
```

Comment:
Not set in stone. Under options for pattern-importer is:

`convertCategoryTitlesDataFile`, which can be changed per-import, but defaults to this included file. This file is used in ./lib/utils/getCategoryPath(). If not file is set, importer just uses the folder names from pattern.yml for each pattern